src/App.js

```diff
@@ -128,7 +128,7 @@ class App extends Component {
       let show_cli_args = query_args.get('show_cli')
 
       document.title = "Tools | Configure a PDB2PQR job";
-      bcrumb = this.createServiceBreadcrumb(['Tools', 'PDB2PQR Job Configuration'])
+      bcrumb = this.createServiceBreadcrumb(['Tools', 'PDB2PQR (v3.1.0) Job Configuration'])
```

Comment:
Can the v3.1.0 be retrieved from the environment or at least a configuration file?

Reply:
Updated the code to do so now.  The defaults present `env/.env` are what we're gonna use, but we can always override at deployment